Documentation/ceph-object-bucket-claim.md

```diff
@@ -102,18 +102,79 @@ metadata:
   labels:
     aws-s3/object [1]
 provisioner: rook-ceph.ceph.rook.io/bucket [2]
-parameters: [3]
+parameters:
   objectStoreName: my-store
   objectStoreNamespace: rook-ceph
-  region: us-west-1
+  region: us-west-1 [3]
   bucketName: ceph-bucket [4]
 reclaimPolicy: Delete [5]
 ```
 1. `label`(optional) here associates this `StorageClass` to a specific provisioner.
 1. `provisioner` responsible for handling `OBCs` referencing this `StorageClass`.
-1. **all** `parameter` required.
+1. `region`(optional) defines the region in which the bucket should be created. For RGW, this maps to [zonegroup](https://docs.ceph.com/en/latest/radosgw/multisite/#zone-groups) of the server. If user does not define `region`, Rook will fill this value accordingly. Please check example at the end of the doc.
```

Comment:
This is still not very clear. What is the region set to if the user doesn't define it? "Accordingly" isn't clear enough.

Reply:
@BlaineEXE : earlier version was `Rook Operato fill this value from object store context internally`. But @travisn suggested it better not to give internal code details for the end-user. Hence changed like this. I can revert back the change

Reply:
Even the text "fill this value from object store context internally" doesn't explain to the user what the value will be. In order for the user to understand when they need to modify the value, they must understand what the default value is. In words good for a user, what value will be the default?

This is also assuming that `region` is even a necessary parameter to have still.

Reply:
Okay will update it again next push